plugins/modules/redhat_subscription.py

```diff
@@ -155,6 +155,16 @@
         default: []
         type: list
         elements: raw
+    keep_consumed_pools:
+        description:
+            - |
+              When using I(pool_ids), f set to C(true), keep the currently consumed subscriptions.
+              When set to C(false), delete all currently consumed subscriptions from the system and consume
+              the subscriptions pool IDs defined in I(pool_ids).
+              It is set to C(false) by default to keep the historical behavior of the module.
```

Comment:
hmm i'm not totally convinced by this doc text: even though i know what's going on, i find it slightly potentially confusing; my take on this:

> This option applies when specifying pools using I(pool_ids) or I(pool), and there are already subscriptions consumed whose pool IDs are not the specified ones (in I(pool_ids) or I(pool)); the default behaviour of the module is to remove those subscriptions, leaving only the specified pools consumed in the system. Setting this option to C(true) changes the behaviour, so that no pool is removed from the system.